GitUI/UserControls/RevisionGrid/Graph/RevisionGraphRevision.cs

```diff
@@ -89,8 +95,8 @@ public int EnsureScoreIsAbove(int minimalScore)
 
         public ObjectId Objectid { get; set; }
 
-        public ConcurrentBag<RevisionGraphRevision> Parents { get; }
-        public ConcurrentBag<RevisionGraphRevision> Children { get; }
+        public ImmutableStack<RevisionGraphRevision> Parents => _parents;
```

Comment:
Is it a personal preference not to use the property backing value as before?

Reply:
➡️ No, the backing field is no longer `readonly`.